roles/cadvisor/meta/argument_specs.yml

```
@@ -64,6 +64,18 @@ argument_specs:
         default: []
         type: "list"
         choices: *metrics_choices
+      cadvisor_store_container_labels:
+        description: "do not store all container labels"
+        type: "bool"
+        default: false
```

Shouldn't this be
```suggestion
        default: true
```
as defined in `defaults/main.yml`? Or should `defaults/main.yml` perhaps be set to `false`?

---

Sorry, you are correct. Updating description too